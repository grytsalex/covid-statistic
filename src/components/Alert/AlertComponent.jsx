import React from "react";
import { AlertMessage, AlertWrapper } from "./styledComponent";
import PropTypes from "prop-types";

export const AlertComponent = ({ message }) => {
  return (
    <AlertWrapper>
      <AlertMessage>{message}</AlertMessage>
    </AlertWrapper>
  );
};

AlertComponent.propTypes = {
  message: PropTypes.string,
};
