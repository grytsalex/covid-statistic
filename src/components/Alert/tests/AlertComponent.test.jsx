import React from "react";
import { AlertComponent } from "../AlertComponent";
import { shallow, mount } from "enzyme";

jest.mock("../styledComponent", () => ({
  AlertMessage: (props) => <div className="alert-message" {...props} />,
}));

describe("Alert", () => {
  const commonProps = {
    message: "message",
  };

  it("should match snapshot", () => {
    expect(shallow(<AlertComponent {...commonProps} />)).toMatchSnapshot();
  });
});
