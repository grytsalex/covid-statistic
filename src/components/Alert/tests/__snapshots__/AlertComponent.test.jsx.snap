// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert should match snapshot 1`] = `
<styled.div>
  <styled.h1>
    message
  </styled.h1>
</styled.div>
`;
